src/schemas/blog_post.json

```diff
@@ -1,236 +1,246 @@
 {
-  "Main" : {
```

Comment:
I think Prismic changed the format of their JSON files or something. This diff is much, much smaller if you ignore whitespace.